framework/egl/wrapper/eglwEnums.inl

```diff
@@ -238,6 +238,7 @@
 #define EGL_LOSE_CONTEXT_ON_RESET_EXT						0x31BF
 #define EGL_PLATFORM_X11_EXT								0x31D5
 #define EGL_PLATFORM_X11_SCREEN_EXT							0x31D6
+#define EGL_PLATFORM_WAYLAND_KHR							0x31D8
```

Comment:
Is this manually added? The file is generated by `scripts/gen_egl.py` and running it again would make this disappear. Instead, you should add relevant ext to `scripts/egl/common.py` to have enum value added here automatically.